mozilla-central-mappings.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@jasonLaster I put this under the root right now, maybe we should have a firefox-integration folder or something?